server/services/prisonerProfileService.test.ts

```diff
@@ -449,21 +450,19 @@ describe('Prisoner profile service', () => {
         ],
         numberOfElements: 1,
       }
-      const prisonerProfile = createInmateDetail()
 
-      const inmateDetail = <InmateDetail>{
-        offenderNo: prisonerProfile.offenderNo,
-        firstName: prisonerProfile.firstName,
-        lastName: prisonerProfile.lastName,
-        dateOfBirth: prisonerProfile.dateOfBirth,
+      const inmateDetail = createInmateDetail({
         activeAlertCount: 4,
         inactiveAlertCount: 1,
-        legalStatus: 'REMAND',
         alerts: [inactiveAlert, nonRelevantAlert, ...alertsToFlag],
-      }
+        legalStatus: 'REMAND',
+      })
+
+      const prisoner = createPrisoner()
```

Comment:
Or maybe use the prisoner object as the expected results parts? Otherwise if you change the DoB in the standard prisoner you have to change it every test that checks it, if you say it should equal `prisoner.dob` or similar you don't need to do that? Or some similar way of doing that?